dowhy/gcm/falsify.py

```diff
@@ -152,42 +144,48 @@ def validate_lmc(
     )
 
     # Gather results
-    p_values_memory.clear_placeholders()  # Clear placeholders
     for i, (node, non_desc, parents) in enumerate(to_test):
         p_values_memory.add_p_value(p_values[i], node, non_desc, parents)
 
     # Summarize
     for node, non_desc, parents in triples:
-        validation_summary[FALSIFY_N_TESTS] += 1
         lmc_p_value = p_values_memory.get_p_value(node, non_desc, parents)
-        validation_summary[FALSIFY_P_VALUES][(node, non_desc)] = (lmc_p_value, lmc_p_value <= significance_level)
-        if lmc_p_value <= significance_level:
-            validation_summary[FALSIFY_N_VIOLATIONS] += 1
+        if lmc_p_value != -1:
+            validation_summary[FalsifyConst.N_TESTS] += 1
+            validation_summary[FalsifyConst.P_VALUES][(node, non_desc)] = (
+                lmc_p_value,
+                lmc_p_value <= significance_level,
+            )
+            if lmc_p_value <= significance_level:
+                validation_summary[FalsifyConst.N_VIOLATIONS] += 1
     return validation_summary
 
 
-def validate_parental_dsep(
-    causal_graph: DirectedGraph, causal_graph_reference: DirectedGraph, include_unconditional: bool = True, **kwargs
+def validate_tpa(
```

Comment:
What does `tpa` stand for? Isn't `dsep` already a good hint what this method validates?

Reply:
`tpa` stands for parental triple. This is how we call it in the paper and I think this unifies it a bit. I explained it in the docstring now!